src/game/state.py

```diff
@@ -0,0 +1,53 @@
+from .turn import Turn
+from .tools import GRID_SIZE, PIECES_NUMBER, get_coordinates
+
+
+class State:
+    """Definition of all data of the game at an instant:
+    - grid [2 dimension list]
+    - remaining_pieces [list of numbers (from 1 to 16)]
+    - game_turn [GameTurn]"""
+
+    def __init__(self, initial_state=""):
+        self.grid = self.init_grid()
+        self.remaining_pieces = self.init_remaining_pieces()
+        self.game_turn = Turn()
+        self.message = ""
+
+    def load_state(self, initial_state):
```

Comment:
This function name is not talking to me. Can you use another name like "fromJson" or from "fromObject" (I don't know what data type is initial_state)